file_server.go

```diff
@@ -29,8 +30,26 @@ func startFileServer() {
 	os.MkdirAll(opt.metaDir, os.ModePerm)
 	http.HandleFunc("/", fileServer)
 
-	log.Printf("Serving on HTTP port 4000\n")
-	log.Fatal(http.ListenAndServe(":4000", nil))
+	port := strconv.Itoa(opt.port)
```

Comment:
See discussion about not having an extra port parameter on PR review comments :)